Sources/protoc-gen-grpc-swift/options.swift

```diff
@@ -143,13 +143,13 @@ final class GeneratorOptions {
         } else {
           throw GenerationError.invalidParameterValue(name: pair.key, value: pair.value)
         }
-    
-      case "ReflectionService":
-          if let value = Bool(pair.value) {
-              self.reflectionService = value
-          } else {
-              throw GenerationError.invalidParameterValue(name: pair.key, value: pair.value)
-          }
+
+      case "ReflectionData":
+        if let value = Bool(pair.value) {
+          self.reflectionService = value
```

Comment:
Let's call the property `generateReflectionData`, this aligns with the "Client" and "Server" options mapping to `generateClient` and `generateServer` properties.